terraform/modules/database/rds.tf

```diff
@@ -2,8 +2,23 @@ resource "aws_db_subnet_group" "cluster" {
   subnet_ids = var.vpc.aws_subnets_private[*].id
 }
 
+resource "aws_kms_key" "aurora" {
```

Comment:
deletion_window_in_days = 7 will help to quiet some of the noise from AWS Config if this key needs to be deleted.

Reply:
Done.